Mathlib/Order/Category/NonemptyFinLinOrd.lean

```diff
@@ -53,51 +53,51 @@ instance (α : Type _) [NonemptyFinLinOrd α] : NonemptyFinLinOrd αᵒᵈ :=
   { OrderDual.fintype α with }
 
 /-- The category of nonempty finite linear orders. -/
-def NonemptyFinLinOrdCat :=
+def NonemptyFinLinOrd :=
   Bundled NonemptyFinLinOrd
```

Comment:
I think this is the reason the whole scheme is not a [useless use of `cat`](https://en.wikipedia.org/wiki/UUOC); the suffix helps with disambiguating a few things.

Reply:
You'll notice that `NonemptyFinLinOrd` is an auxiliary definition in the construction of `NonemptyFinLinOrdCat`. I'll rename it.